### PR TITLE
for the update of the main branch after the login page rework

### PR DESCRIPTION
**Pull request by Dobby-progam**

### CODE REVIEW

**Comment by Dobby-progam** [comment]
for the Dobbie, that one if just me but with a diffrent signing, due to local git issues 
you can merge it fine

**Comment by JoshuaSlui** [comment]
Please fix the merge conflict (by keeping the incoming file), and let me know whether or not the PR is ready.
I will check it when you have done this.

File: index.php [hunk 1]
```diff
@@ -32,15 +32,15 @@
                     <span>We are not an official site, and we are not affiliated with Aternos, Aternos GmbH, exaroton and it's associates and employees.</span>
                 </div>
             </div>
-            <p>Welcome to Aternos Guides, an article site ran by Aternos regulars and an Aternos moderator. Here you may find all kinds of articles about topics and issues that are not covered in Aternos' help center itself. Have an issue that you don't know the fix to? Try our articles below, in the articles page or join the Aternos discord for help.</p>
+            <p><b>Welcome to Aternos Guides</b><br> an article site ran by Aternos regulars and an Aternos moderator. Here you may find all kinds of articles about topics and issues that are not covered in Aternos' help center itself. Have an issue that you don't know the fix to? Try our articles below, in the articles page or join the Aternos discord for help.</p>
```
Comment (JoshuaSlui) [inline index.php]: Please don't use breaks in text like this. It makes the text look unnatural and you should avoid using these in text as much as possible.

File: register.php [hunk 2]
```diff
@@ -15,29 +15,44 @@
...
-        <input type="email" placeholder="example@gmail.com" name="mail" id="mail" required>
-
-        <label for="psw"><b>Password</b></label>
-        <input type="password" placeholder="Enter Password" name="psw" id="psw" required>
-        <?php
-        if (isset($_SESSION['exCode'])) {
-            echo '<p class="text-warning">' . $_SESSION['exCode'] . '<br>';
-        }
-        session_destroy()
-        ?>
```
Comment (JoshuaSlui) [inline register.php:34]: You should keep the php script here. You seem to have removed it entirely.

**Review by JoshuaSlui — CHANGES_REQUESTED** [review]

File: index.php [hunk 3]
```diff
@@ -32,15 +32,15 @@
                     <span>We are not an official site, and we are not affiliated with Aternos, Aternos GmbH, exaroton and it's associates and employees.</span>
                 </div>
             </div>
-            <p>Welcome to Aternos Guides, an article site ran by Aternos regulars and an Aternos moderator. Here you may find all kinds of articles about topics and issues that are not covered in Aternos' help center itself. Have an issue that you don't know the fix to? Try our articles below, in the articles page or join the Aternos discord for help.</p>
-
+            <p><b>Welcome to Aternos Guides</b> </p>
+            <p>An article site ran by Aternos regulars and an Aternos moderator. Here you may find all kinds of articles about topics and issues that are not covered in Aternos' help center itself. Have an issue that you don't know the fix to? Try our articles below, in the articles page or join the Aternos discord for help.</p>
```
Comment (JoshuaSlui) [inline index.php]: I am still not fond of separating the welcome part from the about part itself. It still feels and looks unnatural.

**Review by JoshuaSlui — CHANGES_REQUESTED** [review]

File: register.php (hunk 2 above)
Comment (Dobby-progam) [reply register.php:34]: fixed in [0b525cd](https://github.com/BlueAtomic/Aternos-guides/pull/3/commits/0b525cd67d96886b405c3e6f603e00de100a7b48)

File: index.php (hunk 1 above)
Comment (Dobby-progam) [reply index.php]: fixed in [1e02d11](https://github.com/BlueAtomic/Aternos-guides/pull/3/commits/1e02d1167098a0039092e7a21a3e9bc00972293b)

File: index.php (hunk 3 above)
Comment (Dobby-progam) [reply index.php]: fiexed in [92ea1b3](https://github.com/BlueAtomic/Aternos-guides/pull/3/commits/92ea1b3bfb4c5230d958c1a968f1a5f158baae06)

**Review by JoshuaSlui — APPROVED** [review]

**Pull request merged**